code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -36,17 +36,19 @@
 	var/blood_volume = 560                               // Initial blood volume.
 	var/hunger_factor = 0.05                             // Multiplier for hunger.
 
+	var/taste_sensitivity = TASTE_NORMAL                 // How sensitive the species is to minute tastes.
```

Comment:
Looks like there's a whole new 'taste' system about tasting reagents when drinking, might want to include that in the changelog.